test/functional/Jsr292/src/com/ibm/j9/jsr292/LookupAPITests_Find.java

```diff
@@ -2106,7 +2106,15 @@ public void test_FindSpecial_Public_CrossPackage_InnerClass() throws Throwable {
 	 */
 	@Test(groups = { "level.extended" })
 	public void test_FindSpecial_Default_CrossPackage_Interface() throws Throwable {
-		MethodHandle example = MethodHandles.lookup().findSpecial(CrossPackageDefaultMethodInterface.class, "addDefault", MethodType.methodType(int.class, int.class, int.class), CrossPackageDefaultMethodInterface.class);
+		try {
+			MethodHandles.lookup().findSpecial(CrossPackageDefaultMethodInterface.class, "addDefault", MethodType.methodType(int.class, int.class, int.class), CrossPackageDefaultMethodInterface.class);
```

Comment:
We should improve readability by simplifying the code.
```suggestion
			Class<?> cls = CrossPackageDefaultMethodInterface.class;
			MethodType mt = MethodType.methodType(int.class, int.class, int.class);
			MethodHandles.lookup().findSpecial(cls, "addDefault", mt, cls);
```

Reply:
Should I just do this for the whole file too?

Reply:
No, just the impacted method/test-case should be fine, but it can also be left as-is for consistency with the rest of the file. I leave it to your discretion.

Reply:
Changing it is cleaner. I prefer that.